mirrord-operator/templates/namespaced-role.yaml

```diff
@@ -10,6 +10,6 @@ metadata:
           {{- toYaml (index $.Values.role "mirrord-operator-user" "labels") | nindent 4 }}
     {{- end }}
 rules:
-{{- include "mirrord-operator.rules" . | nindent 0 }}
+{{- include "mirrord-operator.rules" $ | nindent 0 }}
```

Comment:
This is a small bug, `.` refers to the range object which is a string value (namespace name). `$` refers to the root value object.